doc/content.tex

```diff
@@ -1,1347 +0,0 @@
-         l_t = \alpha y_t + (1 - \alpha) (l_{t-1} + b_{t-1}) \\ \nonumber
-         b_t = \beta (l_t - l_{t-1}) + (1 - \beta)b_{t-1} 
-    \end{gather}
-
-    Parameter $b_t$ denotes a slope of the series and the parameter $l_t$ level. There is also a new smoothing
-    parameter for the slope\,--\,$\beta$. Its rage is equal to $\alpha$, so $\alpha,\beta \in \interval[{0,1}]$.
-
-    %%%%%%%%
-    \section{Holt-Winters Seasonal Model} \label{sec:triple-ex}
-    This model is an extension of Holt's linear trend method with added seasonality. It is also called triple
-    exponential smoothing. In this model there are three equations \ref{eq:holt-winters}. One for level, second for
-    trend and third for seasonality. Each pattern uses smoothing constant $ \alpha,\beta,\gamma \in \interval[{0,1}]$.
-
-    \begin{gather} \label{eq:holt-winters}
-        \hat{y}_{t+h|t} = l_{t} + hb_{t} + s_{t+h_m-m}\\ \nonumber
-        l_t = \alpha (y_t - s_{t-m}) + (1 - \alpha) (l_{t-1} + b_{t-1}) \\ \nonumber
-        b_t = \beta (l_t - l_{t-1}) + (1 - \beta)b_{t-1} \\ \nonumber
-        s_t = \gamma (y_t - l_{t-1} - b_{t-1}) + (1-\gamma)s_{t-m}
-    \end{gather}
-
-    Where $h_m=[(h-1) \mod m] + 1$, which ensures that the estimates of the seasonal indices came from the correct
-    season. This model can be used only if the period of time series is know beforehand. In Hawkular the period  of
-    the time series is unknown, therefore period identification should be also implemented.
-
-    Each exponential smoothing model contains parameters which should be precisely estimated ($\alpha, \beta,
-    \gamma$). But it is also possible to use default values from literature. However, with default values models produce
-    higher forecasting error than with estimated parameters. Estimation process defines objective function which
-    value can be mean squared error, mean absolute error or likelihood. Then this objective function is passed to
-    non-linear optimization algorithm.
-
-    Time complexity of exponential smoothing models for learning $N$ observations is $O(N)$. With enabled optimization
-    the complexity depends on used optimization algorithm.
-
-    Exponential smoothing models have advantage over other more complicated models that if a system needs to achieve high
-    performance than default parameters can be used. This would not be possible for instance with ARIMA models.
-
-    %%%%%%%%
-    \section{Box\,--\,Jenkins Methodology (ARIMA)} \label{sec:arima}
-    Models from Box\,--\,Jenkins methodology are the most widely used in time series analysis for econometric data.
-    This methodology is based on analysis of autocorrelation (ACF) and partial autocorrelation (PACF) functions.
-    
-    The most generic model is ARIMA(p, d, q). It combines together autoregressive, integrated and
-    moving average parts. An autoregressive model (AR) consist of sum of weighed lagged observations \ref{eq:ar-model}.
```

Comment:
It combines autoregressive, integrated and moving average parts together . /slovosled